src/components/modern/FeeLabel.js

```diff
@@ -106,9 +105,6 @@ const LabelWrapper = styled.View`
   align-items: center;
 `;
 
-const FeePill = styled.TouchableOpacity`
+const FeeValue = styled.TouchableOpacity`
   justify-content: center;
-  padding: 0 ${spacing.small}px;
-  border-radius: 12px;
-  background-color: ${({ theme, $isNotEnough }) => ($isNotEnough ? theme.colors.negative : theme.colors.labelTertiary)};
```

Comment:
You probably need to keep some coloring for enough/not-enough fee value.

Reply:
@paragramm what will be the coloring for enough/not-enough fee value?

Reply:
![button](https://user-images.githubusercontent.com/66996635/128485788-7d35f47e-45f3-4c1a-93b7-208e5a290975.png)

Reply:
@divyaBacancy please refer to Zeplin https://zpl.io/adKQlYn